letsencrypt/rootfs/etc/cont-init.d/file-structure.sh

```diff
@@ -7,82 +7,82 @@ mkdir -p /data/workdir
 mkdir -p /data/letsencrypt
 
 # Setup Let's encrypt config
-echo -e "dns_desec_token = $(bashio::config 'dns.desec_token')\n" \
+echo -e "aws_access_key_id = $(bashio::config 'dns.aws_access_key_id')\n" \
+      "aws_secret_access_key = $(bashio::config 'dns.aws_secret_access_key')\n" \
+      "dns_cloudns_auth_password = $(bashio::config 'dns.cloudns_auth_password')\n" \
+      "dns_desec_token = $(bashio::config 'dns.desec_token')\n" \
       "dns_digitalocean_token = $(bashio::config 'dns.digitalocean_token')\n" \
+      "dns_directadmin_password = $(bashio::config 'dns.directadmin_password')\n" \
       "dns_directadmin_url = $(bashio::config 'dns.directadmin_url')\n" \
       "dns_directadmin_username = $(bashio::config 'dns.directadmin_username')\n" \
-      "dns_directadmin_password = $(bashio::config 'dns.directadmin_password')\n" \
       "dns_dnsimple_token = $(bashio::config 'dns.dnsimple_token')\n" \
       "dns_dnsmadeeasy_api_key = $(bashio::config 'dns.dnsmadeeasy_api_key')\n" \
       "dns_dnsmadeeasy_secret_key = $(bashio::config 'dns.dnsmadeeasy_secret_key')\n" \
+      "dns_domainoffensive_token = $(bashio::config 'dns.domainoffensive_token')\n" \
```

Comment:
According to [documentation](https://github.com/domainoffensive/certbot-dns-domainoffensive/tree/master) this should contain api as before no?

```suggestion
      "dns_domainoffensive_api_token = $(bashio::config 'dns.domainoffensive_token')\n" \
```